.github/workflows/worker-deploy.yml

```diff
@@ -0,0 +1,30 @@
+name: Deploy
+
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - main
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    name: Deploy
+    steps:
+      - name: Setup Node
+        uses: actions/setup-node@v4
+        with:
+          node-version: "20"
```

Comment:
Always use the correct version 20.10.0

Reply:
https://github.com/ubiquibot/assistive-pricing/pull/11